.github/actions/download-plugin/action.yml

```diff
@@ -17,20 +17,12 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - name: Download OpenSearch for Linux
-      uses: peternied/download-file@v2
-      if: ${{ runner.os == 'Linux' }}
-      with:
-        url: https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/${{ inputs.opensearch-version }}/latest/linux/x64/tar/builds/opensearch/plugins/${{ inputs.plugin-name }}-${{ inputs.plugin-version }}.zip
-
-    - name: Download OpenSearch for Windows
-      uses: peternied/download-file@v2
-      if: ${{ runner.os == 'Windows' }}
-      with:
-        url: https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/${{ inputs.opensearch-version }}/latest/windows/x64/zip/builds/opensearch/plugins/${{ inputs.plugin-name }}-${{ inputs.plugin-version }}.zip
-
-    - name: Rename the Plugin Files
-      run: mv opensearch-security-${{ inputs.plugin-version }}.zip opensearch-security.zip
+    - run: |
```

Comment:
Could we add a `name` for this task, so that we can easy to track in the terminal. The rest looks good to me!